### ISSUE TITLE
Deprecation warning: django.utils.translation.ugettext_lazy() is deprecated

### ISSUE THREAD

**Issue by matthewhegarty** [aaugustin/django-sequences#23]
When running tests with deprecation warnings, I see the following warning:

```bash
python -Wa ./manage.py test 
site-packages/sequences/models.py:19: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  verbose_name_plural = _("sequences")
```

Thanks for creating and maintaining this package.

**Comment by aaugustin** [comment]
Fixed in 631b00dd.

**Comment by aaugustin** [comment]
Thanks for the report!
